proposal.rst

```diff
@@ -166,27 +185,35 @@ Project Details
 
```

Comment:
In this section we're looking for you to dig into the problem, not just superficially sketch out some high level ideas.  We need convincing that you understand the problem, have found an appropriate way to solve it and that you understand that approach and how to implement it.  And we need to understand what the end result is going to look like to judge if your proposed solution is likely to work for users.

You mention "DCM" and "IAM" papers (but haven't cited them so the reader is left unsure exactly which papers you mean), and pointed to two apparently random pieces of software which use the techniques, but none of that really explains what you're actually planning to do, or how it's going to look when implemented within Xapian.